sdk/program/Cargo.toml

```diff
@@ -9,6 +9,7 @@ repository = { workspace = true }
 homepage = { workspace = true }
 license = { workspace = true }
 edition = { workspace = true }
+rust-version = "1.68.0" # solana platform-tools rust version
```

Comment:
1. `rustc_version` is the name of a crate: https://crates.io/crates/rustc_version
2. `rust-toolchain.toml` is for the entire monorepo, and i think it even causes `cargo` to select the specified toolchain. whereas the `rust-version = "blah"` field in a Cargo.toml just indicates the minimum supported rustc version for that crate. Ideally these two would be the same, but we currently don't always update the SBF toolchain to the latest stable rust. Looks like Dmitri generally prefers to update to every 2-3 release since it can be long manual process. It would also suck if we held off Rust upgrades for the entire monorepo until the SBF toolchain picked up a new version, so the version difference is something of a compromise between 
3. Best source of truth currently is @dmakarov I guess. But there's also https://github.com/solana-labs/platform-tools/releases/tag/v1.37, which ideally would include info on what Rust version it's based on in the release notes I guess. You can also run `rustc --version` with the release binaries to discover that info, which is exactly what `cargo build-bpf --version` does internally